scripts/eessi_sets.yml

```diff
@@ -7,7 +7,7 @@ eessi_archs:
 #  - macos-x86_64
 
 eessi_sets:
-  - name: eessi-2023.03
+  - name: eessi-2023.04
```

Comment:
The same changes (removing the editors, htop, and lmod overlay/version) also have to be made in this file. Actually, you can/should only make them in this file, and then simply run [generate_set_files.py](https://github.com/EESSI/gentoo-overlay/blob/main/scripts/generate_set_files.py), see https://github.com/EESSI/gentoo-overlay/tree/main/scripts. This will automatically generate the set files for all architectures.